app/components/agent/Index/quickStart.js

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import Relay from 'react-relay';
+import classNames from 'classnames';
+
+import Emojify from '../../shared/Emojify';
+import Panel from '../../shared/Panel';
+
+import 'highlight.js/styles/atom-one-light.css';
+
+// Grab all the guide documents (*.mdx) from the file system (at ../../docs)
+const GUIDES = ((guideRequire) =>
+  guideRequire.keys().map((guidePath) =>
+    guideRequire(guidePath).default
+  )
+)(require.context('../../docs', true, /\.mdx$/));
+
+class QuickStart extends React.Component {
+  static propTypes = {
+    organization: React.PropTypes.shape({
+      agentTokens: React.PropTypes.shape({
+        edges: React.PropTypes.array.isRequired
+      })
+    }).isRequired,
+    relay: React.PropTypes.object.isRequired
+  };
+
+  state = {
+    selectedGuide: null
+  };
+
+  componentDidMount() {
+    this.props.relay.setVariables({ isMounted: true });
+  }
+
+  handleSelectedGuideChange(selectedGuide, evt) {
+    evt.preventDefault();
+    const newState = { selectedGuide: null };
+
+    if (selectedGuide !== this.state.selectedGuide) {
+      newState.selectedGuide = selectedGuide;
+    }
+
+    this.setState(newState);
+  }
+
+  renderGuideButtons() {
+    return (
+      <div className="center" style={{ margin: -5 }}>
+        {
+          GUIDES.map((Guide, index) => (
+            <a
+              key={index}
+              href="#"
+              className={classNames(
+                'inline-block blue hover-navy text-decoration-none border rounded m1 p1',
+                {
+                  'border-white': index !== this.state.selectedGuide,
+                  'border-gray': index === this.state.selectedGuide
+                }
+              )}
+              onClick={this.handleSelectedGuideChange.bind(this, index)} // eslint-disable-line react/jsx-no-bind
```

Comment:
ESLint complains about the use of `bind`, but I don’t see a cleaner way to do this.